DDDEastAnglia/Areas/Admin/Models/SponsorModel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Web;
+
+namespace DDDEastAnglia.Areas.Admin.Models
+{
+    public class SponsorModel
```

Comment:
:+1: See my other comment about confusion as to the intended purpose of `Sponsor`/`SponsorModel`.